pkg/webui/components/header-v2/header-v2.styl

```
@@ -0,0 +1,50 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.container
+  padding: 0 $cs.l
+  background: $c['tts-primary-050']
+  z-index: $zi.nav
+  border-bottom: 1px solid $c-divider
+  display: flex
+  justify-content: space-between
+  align-items: center
+
+.buttons
+  display: flex
+  align-items: center
+
+.buttons > &:not(:last-child)
+  margin-right: $cs.s
+
+.bookmark-button
+  +media-query($bp.s)
+    display: none
+
+.breadcrumbs
+  +media-query($bp.s)
+    display: none
+
+.mobile-menu
+  display: flex
+  align-items: center
+  gap: 7px
```

```suggestion
  gap: $cs.xs
```

Please avoid pixel values, use either relative `rem` values or the scale variables, in this case `$cs.m`.

Also: `.d-flex .al-center .gap-cs-xs`